src/devices/lcec_epocat.c

```diff
@@ -461,67 +493,118 @@ static void lcec_epocat_read(struct lcec_slave *slave, long period) {
   raw_counts[4] = EC_READ_U16(&pd[hal_data->off_ENC + 8]);
 
   // check for index pulse detected
-  if (EC_READ_BIT(&pd[hal_data->off_FLAG], 0) == 1) {
-    // cancel index-enable
-    *(hal_data->enc_00_index_enable) = 0;
+  if (EC_READ_BIT(&pd[hal_data->off_FLAG], 0) == 1)
+  {
+	  // cancel index-enable
+	  *(hal_data->enc_00_index_enable) = 0;
 
-    raw_latch_counts[0] = EC_READ_U16(&pd[hal_data->off_LATCH]);
-    counts_latch[0] = counts[0];
-    counts_latch[0] += (int16_t)(raw_latch_counts[0] - raw_counts_old[0]);
-  }
-  if (EC_READ_BIT(&pd[hal_data->off_FLAG], 1) == 1) {
-    // cancel index-enable
-    *(hal_data->enc_01_index_enable) = 0;
+	  raw_latch_counts[0] = EC_READ_U16(&pd[hal_data->off_LATCH]);
+	  //counts_latch[0] = counts[0];
```

Comment:
I'm not quite sure what's going on here; is the `counts_latch` code supposed to come back at some point?  Since you deleted the definition above, I'd rather see this code deleted instead of just being commented out.  If it's here as a reminder for something, maybe replace it with a `// TODO:` comment instead?

Reply:
Sorry, yes, please delete the commented lines, also in all other parts of the code where not necessary.